pxr/usd/usdSkel/doxygen/schemas.dox

```diff
@@ -530,12 +536,51 @@ using the UsdSkelBindingAPI. The _jointIndices_ primvar provides an array giving
 the joint index of an influence, while the _jointWeights_ primvar provides a
 weight value corresponding to each of those indices.
 
+For example:
```

Comment:
This was a missing example. I made it up based on the explanation paragraph below.